src/app/menu/menu.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="col-md-12 text-center"><h2 class="stage-header">{{language?.menu.heading}}</h2></div>
+<div class="col-md-5 pull-right"><h2 class="stage-header">{{language?.menu.heading}}</h2></div>
```

Comment:
Thanks for working on this. Looks good. But it would be good to keep just this change in this PR because that is what corrects the corresponding issue.

Reply:
But without the other changes, the app cannot be built properly.

Reply:
Yes, but once the other PR which is correcting the build issue gets merged, this won't be needed anymore. I'll get back to you after the previous PR gets merged correctly. :)